driver/src/logging.rs

```diff
@@ -62,9 +62,7 @@ impl<D: Decorator> Drain for CustomFormatter<D> {
             write!(decorator, " ")?;
 
             decorator.start_msg()?;
-            write!(decorator, "{}", record.msg())?;
-
-            write!(decorator, "\n")?;
```

Comment:
I'm glad that this new line is gone, but wonder how this was related to clippy warnings?

Reply:
Clippy was complaining that `write!(decorator, "\n")?;` should be `writeln!(decorator)?;`.